Solutions/RubrikSecurityCloud/Playbooks/RubrikRetrieveUserIntelligenceInformation/azuredeploy.json

```diff
@@ -0,0 +1,1628 @@
+                                            "variables": {
+                                                "sid": "@{variables('userId')}",
+                                                "timelineDate": "@{concat(formatDateTime(utcNow(),'yyyy-MM-dd'),'T00:00:00+00:00')}"
+                                            }
+                                        },
+                                        "headers": {
+                                            "Authorization": "Bearer @{variables('AccessToken')}"
+                                        },
+                                        "method": "POST",
+                                        "uri": "@{variables('BaseUrl')}/api/graphql"
+                                    }
+                                },
+                                "Success_Response": {
+                                    "runAfter": {
+                                        "Does_policy_hits_for_user_request_get_status_code_equal_to_200": [
+                                            "Succeeded"
+                                        ]
+                                    },
+                                    "type": "Response",
+                                    "kind": "Http",
+                                    "inputs": {
+                                        "body": "Comments are added to incident related to Risk Details and Policy Hits Data for user @{variables('userId')}",
+                                        "statusCode": 200
+                                    }
+                                }
+                            },
+                            "runAfter": {
+                                "Initialize_user_policy_hits_data_logType": [
+                                    "Succeeded"
+                                ]
+                            },
+                            "else": {
+                                "actions": {
+                                    "Add_comment_to_incident_(V3)": {
+                                        "runAfter": {},
+                                        "type": "ApiConnection",
+                                        "inputs": {
+                                            "body": {
+                                                "incidentArmId": "@triggerBody()?['incidentARMId']",
+                                                "message": "<p>No user with name @{if(empty(variables('Username')),variables('Email'),variables('Username'))} is available.</p>"
```

Comment:
Hello @niralishah-crest, After my investigation I came to find out arm ttk is failing for this line 
With warning message as 
Variable reference is not contained within an expression:

Do you know anything on this

Reply:
@v-prasadboke  This line uses logic apps' variables as reference and not template variables.

Reply:
yes but if you remove it and repackage the solution arm-ttk wont fail

Reply:
In such case I have to look into it for removal, as it is required as part of adding comment to the incident.

Reply:
What if we use text field instead of message

Reply:
@v-prasadboke from UI, I have added incident comment which is added as message field value in backend.